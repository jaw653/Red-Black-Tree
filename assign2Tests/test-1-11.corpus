"o	f jmv	
bs"
"miamb	

 hj"
"
 y
u	st 	z"
" 	d	g
t
as
"
"ndlt  s tyr"
"k m vsocyvr"
"b ep
mollqe"
"v	zdyodxucb"
"
azk	 
se h"
"lf
z	vac	de"
"
t 	fy	fkxg"
"
g
jqt
y
rf"
"hk sy		t	zy"
"fm	 z q	es
"
"gp
 f
rf
 
"
"l	wmduq 	
k"
"cnva	r	vf s"
"l
h		jq  de"
"ulyur	neu e"
"
pxenyu		 	"
"rqnsfjv
kta"
"mntd	amfuwq"
"p	lcu 
bjb
"
"gx f	gc	 he"
"

b
iej wtj"
"
		hf
z
d
o"
" ntaba	
a
i"
" jpq	 b
zxp"
"
ttls	tle	o"
"wp zu l ekw"
"tn	akjvmi g"
"it
z 
 cth
"
" mlp	
a
iv
"
"xp pdksky	 "
"u 	bhymcfvd"
"guev
a	b 	 "
"igetf b jpf"
"


h	k	p
hl"
"tnk
x

  f "
"q	qs
jm 
f	"
"up

i
ftg o"
"	
ysj
ir
pp"
"n ad	j z		h"
" t li uajmq"
" lwt vi bzm"
"
lnqc s 
uw"
"qkfhax
qu	k"
"q
 n
	
lqh	"
"
wmt	 	n zf"
"a
i		
gap
n"
" sr
ugb qhh"
"e mgiekq   "
"jtgpsrmt	
z"
"xqkl	pnm	f
"
"zrlty	 dco	"
"udlz	gdwi	
"
"nl
qa sj w "
"

	gks a	pn"
"gjwaihteun "
" qhcv	bq ot"
"	awjjczjpd "
"s
  hxby 
s"
"l 
	 xjdc
v"
" ggwb	cieyi"
"sruimnnqn	j"
"b	qnbw	ql	e"
"	thibzco xg"
"qwu	swg
 es"
"ls inkgah	x"
"sqpri
 f	ac"
"m 	p
w

atv"
"x	hi  m	qxl"
"yri l
bydio"
" uwsei
 	
 "
" 
cgq
 	  d"
"lf	zjnav
hq"
"

	e	r wgj "
"
s
	rc qdeh"
"h 	
 xc				"
"m
 u
hr ce	"
"l	mn	r 
w	z"
"s 	di
mthy	"
"
l	e
	d  nx"
"xt	
tcmo
d
"
"  nua ptgpj"
"gb	mps	qi a"
"wa	 qn
nmnb"
"g y
 
zvl
 "
"	 jw jhtdew"
"
 
f 
v
f	q"
"zw cadu	qkp"
"ntep 
dx x
"
"atvi	u
omm
"
"	plh vy	x
	"
"kf
wg	pt	k
"
"flgy	xn isw"
"ki			 o	
cv"
"h  suqlh rx"
"
ar

	
l
tl"
"
n	
aoc
huc"
" t
d	iqt  o"
"

gy z	vqhw"
"orfib	  	 k"
"gqo
 v	gluu"
"mc
n	od
	nq"
"	ycv
 umsv
"
"  ed
wladsr"
"fgwum		c
fl"
"qpas
zdk i
"
"
s	ppde

yc"
"j	 w	 as
tw"
"	
izor			 r"
"	dfgs
tq	j
"
"sox	h
mvn j"
"r		

xiuc	
"
"	eni	  urwe"
"qr	bj s yaf"
"
m irir s	f"
"j 
i
ga
jt	"
"b ok	

		hb"
"m cn	k zwhy"
"ze 
ypk  
r"
"ky	yk	rl	wa"
"bwehg
si mj"
