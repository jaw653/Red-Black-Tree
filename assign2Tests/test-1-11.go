s
r
d "o	f jmv	
bs"
d "miamb	

 hj"
d "
 y
u	st 	z"
d " 	d	g
t
as
"
d "k m vsocyvr"
d "v	zdyodxucb"
d "
azk	 
se h"
d "lf
z	vac	de"
d "
t 	fy	fkxg"
d "
g
jqt
y
rf"
d "hk sy		t	zy"
d "fm	 z q	es
"
d "l	wmduq 	
k"
d "cnva	r	vf s"
d "l
h		jq  de"
d "ulyur	neu e"
d "
pxenyu		 	"
d "rqnsfjv
kta"
d "p	lcu 
bjb
"
d "gx f	gc	 he"
d "

b
iej wtj"
d "
		hf
z
d
o"
d " jpq	 b
zxp"
d "
ttls	tle	o"
d "wp zu l ekw"
d "tn	akjvmi g"
d "it
z 
 cth
"
d " mlp	
a
iv
"
d "xp pdksky	 "
d "guev
a	b 	 "
d "igetf b jpf"
d "tnk
x

  f "
d "q	qs
jm 
f	"
d "up

i
ftg o"
d "	
ysj
ir
pp"
d "n ad	j z		h"
d " t li uajmq"
d " lwt vi bzm"
d "
lnqc s 
uw"
d "qkfhax
qu	k"
d "q
 n
	
lqh	"
d "
wmt	 	n zf"
d "a
i		
gap
n"
d " sr
ugb qhh"
d "e mgiekq   "
d "jtgpsrmt	
z"
d "xqkl	pnm	f
"
d "udlz	gdwi	
"
d "nl
qa sj w "
d "

	gks a	pn"
d "gjwaihteun "
d " qhcv	bq ot"
d "	awjjczjpd "
d "s
  hxby 
s"
d "l 
	 xjdc
v"
d " ggwb	cieyi"
d "sruimnnqn	j"
d "b	qnbw	ql	e"
d "ls inkgah	x"
d "sqpri
 f	ac"
d "m 	p
w

atv"
d "x	hi  m	qxl"
d " uwsei
 	
 "
d " 
cgq
 	  d"
d "lf	zjnav
hq"
d "
s
	rc qdeh"
d "h 	
 xc				"
d "m
 u
hr ce	"
d "l	mn	r 
w	z"
d "
l	e
	d  nx"
d "xt	
tcmo
d
"
d "  nua ptgpj"
d "gb	mps	qi a"
d "wa	 qn
nmnb"
d "g y
 
zvl
 "
d "	 jw jhtdew"
d "
 
f 
v
f	q"
d "zw cadu	qkp"
d "ntep 
dx x
"
d "atvi	u
omm
"
d "	plh vy	x
	"
d "kf
wg	pt	k
"
d "flgy	xn isw"
d "ki			 o	
cv"
d "h  suqlh rx"
d "
ar

	
l
tl"
d "
n	
aoc
huc"
d " t
d	iqt  o"
d "

gy z	vqhw"
d "orfib	  	 k"
d "gqo
 v	gluu"
d "mc
n	od
	nq"
d "	ycv
 umsv
"
d "  ed
wladsr"
d "qpas
zdk i
"
d "
s	ppde

yc"
d "	
izor			 r"
d "	dfgs
tq	j
"
d "sox	h
mvn j"
d "r		

xiuc	
"
d "	eni	  urwe"
d "
m irir s	f"
d "j 
i
ga
jt	"
d "b ok	

		hb"
d "m cn	k zwhy"
d "ze 
ypk  
r"
d "ky	yk	rl	wa"
d "bwehg
si mj"
s
r
