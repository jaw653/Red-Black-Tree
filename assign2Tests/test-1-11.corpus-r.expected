0: miamb hj-B(miamb hj-B)-
1: hk sy t zy-B(miamb hj-B)-l t fy fkxg-B(miamb hj-B)-r
2: g jqt y rf-R(hk sy t zy-B)-l k m vsocyvr-B(hk sy t zy-B)-r qkfhax qu k-B(t fy fkxg-B)-l v zdyodxucb-B(t fy fkxg-B)-r
3: b qnbw ql e-B(g jqt y rf-R)-l gp f rf-B(g jqt y rf-R)-r j w as tw-B(k m vsocyvr-B)-l lnqc s uw-R(k m vsocyvr-B)-r o f jmv bs-R(qkfhax qu k-B)-l s hxby s-R(qkfhax qu k-B)-r ttls tle o-B(v zdyodxucb-B)-l xp pdksky-R(v zdyodxucb-B)-r
4: b ep mollqe-B(b qnbw ql e-B)-l e r wgj-R(b qnbw ql e-B)-r gjwaihteun-B(gp f rf-B)-l gx f gc he-B(gp f rf-B)-r it z cth-R(j w as tw-B)-l jtgpsrmt z-R(j w as tw-B)-r l h jq de-B(lnqc s uw-R)-l lwt vi bzm-B(lnqc s uw-R)-r ndlt s tyr-B(o f jmv bs-R)-l pxenyu-B(o f jmv bs-R)-r rqnsfjv kta-B(s hxby s-R)-l sr ugb qhh-B(s hxby s-R)-r tn akjvmi g-R(ttls tle o-B)-l ulyur neu e-R(ttls tle o-B)-r wp zu l ekw-B(xp pdksky-R)-l y u st z-B(xp pdksky-R)-r
5: awjjczjpd-R(b ep mollqe-B)-l b iej wtj-B(b ep mollqe-B)-r d g t as-B(e r wgj-R)-l flgy xn isw-B(e r wgj-R)-r gb mps qi a-B(gjwaihteun-B)-l =gks a pn-B(gjwaihteun-B)-r guev a b-B(gx f gc he-B)-l h xc-R(gx f gc he-B)-r =igetf b jpf-B(it z cth-R)-l izor r-B(it z cth-R)-r =jpq b zxp-B(jtgpsrmt z-R)-l =jw jhtdew-B(jtgpsrmt z-R)-r ki o cv-R(l h jq de-B)-l l wmduq k-R(l h jq de-B)-r =ls inkgah x-B(lwt vi bzm-B)-l m u hr ce-R(lwt vi bzm-B)-r mntd amfuwq-R(ndlt s tyr-B)-l ntaba a i-R(ndlt s tyr-B)-r p lcu bjb-B(pxenyu-B)-l q qs jm f-B(pxenyu-B)-r qwu swg es-R(rqnsfjv kta-B)-l =s di mthy-B(rqnsfjv kta-B)-r s rc qdeh-R(sr ugb qhh-B)-l sruimnnqn j-B(sr ugb qhh-B)-r t li uajmq-B(tn akjvmi g-R)-l =tnk x f-B(tn akjvmi g-R)-r u bhymcfvd-B(ulyur neu e-R)-l up i ftg o-B(ulyur neu e-R)-r wmt n zf-B(wp zu l ekw-B)-l =x hi m qxl-B(wp zu l ekw-B)-r xqkl pnm f-B(y u st z-B)-l ysj ir pp-R(y u st z-B)-r
6: ar l tl-B(awjjczjpd-R)-l =azk se h-B(awjjczjpd-R)-r =b ok hb-R(b iej wtj-B)-r cgq d-B(d g t as-B)-l e mgiekq-B(d g t as-B)-r f v f q-R(flgy xn isw-B)-l =fm z q es-B(flgy xn isw-B)-r =g y zvl-R(gb mps qi a-B)-l =ggwb cieyi-R(gb mps qi a-B)-r =gqo v gluu-R(guev a b-B)-l h k p hl-B(h xc-R)-l =hf z d o-B(h xc-R)-r =j i ga jt-R(izor r-B)-r =kf wg pt k-B(ki o cv-R)-l l e d nx-B(ki o cv-R)-r =l mn r w z-B(l wmduq k-R)-l lf z vac de-B(l wmduq k-R)-r m irir s f-B(m u hr ce-R)-l =mc n od nq-B(m u hr ce-R)-r =mlp a iv-B(mntd amfuwq-R)-l n ad j z h-B(mntd amfuwq-R)-r =nl qa sj w-B(ntaba a i-R)-l nua ptgpj-B(ntaba a i-R)-r =orfib k-R(p lcu bjb-B)-l =plh vy x-R(p lcu bjb-B)-r =q n lqh-R(q qs jm f-B)-l =qhcv bq ot-R(q qs jm f-B)-r qpas zdk i-B(qwu swg es-R)-l =r xiuc-B(qwu swg es-R)-r =s ppde yc-B(s rc qdeh-R)-l sqpri f ac-B(s rc qdeh-R)-r =t d iqt o-R(sruimnnqn j-B)-r =thibzco xg-R(t li uajmq-B)-r =udlz gdwi-R(u bhymcfvd-B)-r =uwsei-R(up i ftg o-B)-r =wa qn nmnb-R(wmt n zf-B)-l =xt tcmo d-R(xqkl pnm f-B)-r yri l bydio-B(ysj ir pp-R)-l zrlty dco-B(ysj ir pp-R)-r
7: =a i gap n-R(ar l tl-B)-l =atvi u omm-R(ar l tl-B)-r =bwehg si mj-R(cgq d-B)-l =cnva r vf s-R(cgq d-B)-r =dfgs tq j-R(e mgiekq-B)-l ed wladsr-B(f v f q-R)-l =fgwum c fl-B(f v f q-R)-r =gy z vqhw-R(h k p hl-B)-l =h suqlh rx-R(h k p hl-B)-r =ky yk rl wa-R(l e d nx-B)-l =l xjdc v-R(lf z vac de-B)-l =lf zjnav hq-R(lf z vac de-B)-r =m cn k zwhy-R(m irir s f-B)-l =m p w atv-R(m irir s f-B)-r =n aoc huc-R(n ad j z h-B)-r =ntep dx x-R(nua ptgpj-B)-l =qr bj s yaf-R(qpas zdk i-B)-r =sox h mvn j-R(sqpri f ac-B)-l =ycv umsv-R(yri l bydio-B)-l =ze ypk r-R(zrlty dco-B)-l =zw cadu qkp-R(zrlty dco-B)-r
8: =eni urwe-R(ed wladsr-B)-r
Words/Phrases: 124
Nodes: 124
Minimum depth: 6
Maximum depth: 9
0: j w as tw-B(j w as tw-B)-
1: fgwum c fl-B(j w as tw-B)-l s di mthy-R(j w as tw-B)-r
2: b ep mollqe-B(fgwum c fl-B)-l h k p hl-B(fgwum c fl-B)-r qr bj s yaf-B(s di mthy-R)-l u bhymcfvd-B(s di mthy-R)-r
3: =e r wgj-R(b ep mollqe-B)-r =gp f rf-R(h k p hl-B)-l ndlt s tyr-R(qr bj s yaf-B)-l =qwu swg es-B(qr bj s yaf-B)-r =thibzco xg-B(u bhymcfvd-B)-l yri l bydio-B(u bhymcfvd-B)-r
4: =mntd amfuwq-B(ndlt s tyr-R)-l =ntaba a i-B(ndlt s tyr-R)-r =zrlty dco-R(yri l bydio-B)-r
Words/Phrases: 16
Nodes: 16
Minimum depth: 3
Maximum depth: 5
